### PR TITLE
add vcpkg-tool

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by wolfv** [comment]
@xhochy I know you're one of the other vcpkg users here :) let me know if you're interested in being a co-maintainer.

VCPKG split their repo into two: the "tool" and the portfiles. I am adding a seperate feedstock for the tool here, and will modify the portfile feedstock (github.com/conda-forge/vcpkg) after to depend on the tool.

Activation will also be in the port files since it's just setting up the path to the portfiles.

File: recipes/vcpkg-tool/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+context:
```
Comment (xhochy) [inline recipes/vcpkg-tool/recipe.yaml:1]: We don't support these here 😛 Can you leave this out for now?

**Comment by xhochy** [comment]
I would pass on being a maintainer here, I'm not really using it currently.

File: recipes/vcpkg-tool/recipe.yaml (hunk 1 above)
Comment (wolfv) [reply recipes/vcpkg-tool/recipe.yaml:1]: sure, I wanted to remove that after checking that the build works.

**Comment by wolfv** [comment]
no worries, was just checking

**Pull request merged**